pkg/onboarding/presentation/config.go

```diff
@@ -115,6 +123,11 @@ func Router(ctx context.Context) (*mux.Router, error) {
 	// Shared authenticated routes
 	openSourcePresentation.SharedAuthenticatedRoutes(h, r)
 
+	// Onboarding-service rest routes
```

Comment:
The comment _//Onboarding service rest routes_
Make it that way. Toa iyo hyphen